tasks/translations.md

```diff
@@ -0,0 +1,132 @@
+# GCI Tasks: Creating translation style guides
+
+## Prerequisites
```

Comment:
I would also add
* Native fluency in the appropriate language.

as a prerequisite